components/Cardlist.js

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+
+export default function Cardlist({ children }) {
+  return <StyledCardList>{children}</StyledCardList>;
+}
+const StyledCardList = styled.ul`
```

Comment:
Your component here is called `Cardlist` but your styled component is called `StyledCard**L**ist`. You could consider renaming the `Cardlist` component to `CardList` to match.

Reply:
???

Reply:
It's about the capitalized L of the word "List". 😄 
You use StyledCard**L**ist (uppercase L) but the component is named Card**l**ist (lowercase L)

Reply:
Did not saw that detail but changed it now.